src/components/SortPopover.jsx

```diff
@@ -0,0 +1,103 @@
+import React from 'react'
+import { Popover, Position, Classes } from '@blueprintjs/core'
+import styled from 'styled-components'
+import { useDispatch } from 'react-redux'
+import {productsSortedNameAtoZ, productsSortedNameZtoA, productsSortedByRecentlyAdded, productsSortedBySoonToExpire} from '../store/productsSlice'
+
+export default function SortPopover() {
+  const dispatch = useDispatch()
+
+  const handleRecentSort = () => {
+    dispatch(productsSortedByRecentlyAdded())
+  }
+  const handleExpireSort = () => {
+    dispatch(productsSortedBySoonToExpire())
+  }
+  const handleNameSortAtoZ = () => {
+    dispatch(productsSortedNameAtoZ())
+  }
+  const handleNameSortZtoA = () => {
+    dispatch(productsSortedNameZtoA())
+  }
+
+  return(
+    <PopoverStyled usePortal={false} position={Position.LEFT_TOP} popoverClassName={Classes.POPOVER_DISMISS}>
+      <ButtonStyled type="button" className="button">
+       <IconStyled
+         className="fas fa-sort-amount-down"
```

Comment:
Welche Bedeutung hat eigentlich das "fas fa..." in deinen classNames?

Reply:
Das sind die default Klassen von der Font Awesome library, woher ich mir die icons genommen habe